configuration/verify-source.yaml

```diff
@@ -0,0 +1,57 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Configuration for verify-source CodeBuild project
+
+Resources:
+  CodeBuildProject:
+    Type: AWS::CodeBuild::Project
+    Properties:
+      Name: verify-source
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      Environment:
+        Type: LINUX_CONTAINER
+        ComputeType: BUILD_GENERAL1_SMALL
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:2.0
+      Source:
+        Type: NO_SOURCE
+        BuildSpec:
+          !Sub
+          "version: 0.2\n\nphases:\n  build:\n    commands:\n      -
+           aws s3 cp s3://exodus-pipeline-artifacts/.whitelist.gpg .whitelist.gpg\n      -
+           gpg --import .whitelist.gpg\n      -
+           git clone https://github.com/$REPO_OWNER/$REPO_NAME.git -b $REPO_BRANCH\n      -
+           cd ./$REPO_NAME\n      -
+           git verify-tag $(git describe --exact-match $COMMIT_ID)\n      -
+           aws s3 cp .whitelist.gpg s3://exodus-pipeline-artifacts/.whitelist.gpg"
```

Comment:
As done in a previous attempt at tag verification, this downloads the whitelist from s3 and, once the commit is verified, updates it with the version provided by the commit (reminder that versioning is enabled in the bucket for history).

We discussed before that, instead of doing this, hard-coding the GPG keys was a better option. However, this doesn't work well here, if at all. I think that, if we were to force it, it would be far too difficult to manage, as very specific formatting/use of control characters would likely be necessary.